### PR TITLE
Pre-emptive fix for portrait images and grid

### LINKED ISSUE #1454: iPhone 6 Plus has "margin" to the right of article body text
**Describe the bug**
Visiting an article on iPhone 6 Plus, it's at just the right size to get the "tablet" view and subsequently looks a bit weird. Have confirmed with Natalie from UX that we won't want a gutter of space here - so we should probably tweak the breakpoint at which that happens.

**To Reproduce**

1. On an iPhone 6 Plus
2. Visit any Simorgh article
3. See the space to the right of the body text.

**Expected behavior**
Body text should flow right to the edge of the screen (apart from 8-16px of 'true guttering').

**Screenshots**

Screen Shot 2019-03-21 at 11 41 45

**Smartphone (please complete the following information):**

- Device: iPhone 6 Plus
- OS: 12
- Browser - Safari

**Additional context**
N/A

- [x] Initially labelled with ["bug"](https://github.com/bbc/simorgh/labels/bug)

### PR DESCRIPTION
**Pull request by ChrisBAshton**
Pre-empts fixing an issue with portrait images (see comments on the PR).

Not intended to be merged into #1501 before that gets merged into `latest`.

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Tests added for new features
- [ ] Test engineer approval
- [ ] I have followed [the merging checklist](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/MERGE_PROCESS.md) and this is ready to merge.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1,4 +1,6 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing
-# The command is `git update-index --assume-unchanged .env` 
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
+SIMORGH_BASE_URL=http://localhost:7080
```
Comment (dr3) [inline .env]: This file needs resetting

**Review by dr3 — CHANGES_REQUESTED** [review]
Can you also update `layoutGridItemSmall` as I think that has the same issue

File: .env (hunk 1 above)
Comment (ChrisBAshton) [reply .env]: Why does this keep happening?! 😫

**Comment by ChrisBAshton** [comment]
> Can you also update layoutGridItemSmall as I think that has the same issue

I don't think it applies there? `layoutGridItemSmall` isn't used on paragraphs (it's only used for images where `height > width` AFAIK) and this is a fix for adding margin to paragraphs on mobile screens.

NB this is what `layoutGridItemSmall` currently looks like (ignore the image, I had to change the fixture data image dimensions but didn't bother changing the src)

Screen Shot 2019-03-26 at 08 42 08

**Comment by ChrisBAshton** [comment]
Talked IRL, understood that we probably don't want the 'spacing' on portrait images and should adjust the class accordingly. Have done so here:

https://github.com/bbc/simorgh/pull/1490/commits/22fc2ed9e8aeb41e4d5de91a0b8d27b1020ab1ad

Video:

ezgif-2-3114ce94cb0a

File: src/app/lib/layoutGrid.js [hunk 2]
```diff
@@ -135,18 +134,17 @@ export const layoutGridItemMedium = css`
 export const layoutGridItemSmall = css`
   ${gelGridMargin}
 
-  @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) {
-    grid-column: 1 / span 6;
+  grid-column: 1 / span 6;
+
+  @media (min-width: ${GEL_GROUP_2_SCREEN_WIDTH_MIN}) and (max-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) {
```
Comment (dr3) [inline src/app/lib/layoutGrid.js:139]: This doesnt match zeplin. At 768px zeplin spans 4 columns but this spans 5

**Review by dr3 — CHANGES_REQUESTED** [review]

File: src/app/lib/layoutGrid.js (hunk 2 above)
Comment (ChrisBAshton) [reply src/app/lib/layoutGrid.js:139]: Let's look at it with UX tomorrow.

**Comment by ChrisBAshton** [comment]
Have raised #1501 - let's get that across the board.

We can look at this PR with UX later but it's not a priority for launch, unlike #1501 is.

**Pull request closed without merging**